ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -167,11 +208,16 @@ workdir="$(mktemp -d)"
 cat "${vpc_info_file}" | jq -c -r '[.subnets[] | select(.name|test("control-plane")) | .name]' | yq-go r -P - >${workdir}/controlPlaneSubnets.yaml
 cat "${vpc_info_file}" | jq -c -r '[.subnets[] | select(.name|test("compute")) | .name]' | yq-go r -P - >${workdir}/computerSubnets.yaml
 
+if [[ "${isOldVersion}" == "true" ]]; then
+    rg_name_line="resourceGroupName: ${resource_group}"
+else
+    rg_name_line="networkResourceGroupName: ${resource_group}"
+fi
+
 cat > "${SHARED_DIR}/customer_vpc_subnets.yaml" << EOF
 platform:
   ibmcloud:
-    resourceGroupName: ${resource_group}
-    networkResourceGroupName: ${resource_group}
+    ${rg_name_line}
```

Comment:
shall we need cover resouceGroupName is not same as networkResourceGroupName?

Reply:
That depends on your test case, that is out of this PR's scope. Whatever, I introduced `CREATE_CLUSTER_RESOURCE_GROUP` in `ibmcloud-provision-resourcegroup` to support that behavior.